week04_approx_rl/seminar_pytorch.ipynb

```diff
@@ -156,24 +156,44 @@
       "outputs": [],
       "source": [
         "s, _ = env.reset()\n",
-        "assert tuple(network(torch.tensor([s]*3, dtype=torch.float32)).size()) == (\n",
-        "    3, n_actions), \"please make sure your model maps state s -> [Q(s,a0), ..., Q(s, a_last)]\"\n",
-        "assert isinstance(list(network.modules(\n",
-        "))[-1], nn.Linear), \"please make sure you predict q-values without nonlinearity (ignore if you know what you're doing)\"\n",
-        "assert isinstance(get_action(network, s), int), \"get_action(s) must return int, not %s. try int(action)\" % (type(get_action(network, s)))\n",
-        "\n",
+        "assert tuple(network(torch.tensor([s] * 3, dtype=torch.float32)).size()) == (\n",
+        "    3,\n",
+        "    n_actions,\n",
+        "), \"please make sure your model maps state s -> [Q(s,a0), ..., Q(s, a_last)]\"\n",
+        "assert isinstance(\n",
+        "    list(network.modules())[-1], nn.Linear\n",
+        "), \"please make sure you predict q-values without nonlinearity (ignore if you know what you're doing)\"\n",
+        "assert isinstance(\n",
+        "    get_action(network, s), int\n",
+        "), \"get_action(s) must return int, not %s. try int(action)\" % (\n",
+        "    type(get_action(network, s))\n",
+        ")\n",
+        "    list(network.modules())[-1], nn.Linear\n",
+        "), \"please make sure you predict q-values without nonlinearity (ignore if you know what you're doing)\"\n",
+        "for eps in [0.0, 0.1, 0.5, 1.0]:\n",
```

Comment:
Something definitely went wrong here — this is not syntactically correct.

Reply:
I've reverted commit with formatting for now in order to avoid unobvious bugs, I'll probably get back to this in the future.